SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/SubscriptionManagerV2.swift

```diff
@@ -364,18 +367,17 @@ public final class DefaultSubscriptionManagerV2: SubscriptionManagerV2 {
 
     // MARK: - User
     public var isUserAuthenticated: Bool {
-        return (try? oAuthClient.currentTokenContainer())?.accessToken != nil
-    }
-
-    private func isUserAuthenticated() async -> Bool {
-        let tokenContainer = try? await getTokenContainer(policy: .local)
-        return tokenContainer != nil
+        userDefault.isUserAuthenticated
```

Comment:
the main change here, it is now reading the userdefaults stored value instead of requesting the real token